src/pudl/metadata/classes.py

```diff
@@ -1038,13 +1116,15 @@ class Resource(Base):
     title: String = None
     description: String = None
     harvest: ResourceHarvest = {}
-    group: Literal["eia", "epacems", "ferc1", "ferc714", "glue", "pudl"] = None
     schema_: Schema = pydantic.Field(alias='schema')
     contributors: List[Contributor] = []
     licenses: List[License] = []
-    sources: List[Source] = []
+    sources: List[DataSource] = []
     keywords: List[String] = []
     encoder: Encoder = None
+    field_namespace: Literal["eia", "epacems",
+                             "ferc1", "ferc714", "glue", "pudl"] = None
+    etl_group: String = None
```

Comment:
I think it would be useful to use a `Literal` here. It would at least help catch typos when adding new resources. I'll go ahead and add this if there's no opposititon